[2022-09-30T14:31:43.798+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: web_da_construcao.colector scheduled__2022-08-21T00:00:00+00:00 [queued]>
[2022-09-30T14:31:43.807+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: web_da_construcao.colector scheduled__2022-08-21T00:00:00+00:00 [queued]>
[2022-09-30T14:31:43.807+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-30T14:31:43.833+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-09-30T14:31:43.834+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-30T14:31:44.310+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): colector> on 2022-08-21 00:00:00+00:00
[2022-09-30T14:31:44.315+0000] {standard_task_runner.py:54} INFO - Started process 100 to run task
[2022-09-30T14:31:44.317+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'web_da_construcao', 'colector', 'scheduled__2022-08-21T00:00:00+00:00', '--job-id', '318', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpl1ddohbz']
[2022-09-30T14:31:44.318+0000] {standard_task_runner.py:83} INFO - Job 318: Subtask colector
[2022-09-30T14:31:44.319+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dag.py
[2022-09-30T14:31:44.784+0000] {task_command.py:384} INFO - Running <TaskInstance: web_da_construcao.colector scheduled__2022-08-21T00:00:00+00:00 [running]> on host 0595de4d802c
[2022-09-30T14:31:44.851+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=web_da_construcao
AIRFLOW_CTX_TASK_ID=colector
AIRFLOW_CTX_EXECUTION_DATE=2022-08-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-21T00:00:00+00:00
[2022-09-30T14:31:44.852+0000] {tasks.py:42} INFO - Defining DEC to export information
[2022-09-30T14:31:44.859+0000] {tasks.py:357} INFO - Reading prep dataframe
[2022-09-30T14:31:44.867+0000] {tasks.py:150} INFO - Instantiating the Browser with the requested settings
[2022-09-30T14:31:44.891+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks.py", line 359, in colector
    browser = config_browser(config)
  File "/opt/airflow/dags/tasks.py", line 163, in config_browser
    desired_capabilities=capabilities)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 270, in __init__
    self.start_session(capabilities, browser_profile)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 363, in start_session
    response = self.execute(Command.NEW_SESSION, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 428, in execute
    self.error_handler.check_response(response)
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/errorhandler.py", line 243, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.SessionNotCreatedException: Message: create container: Error response from daemon: No such image: selenoid/vnc:firefox_72.0

[2022-09-30T14:31:44.906+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=web_da_construcao, task_id=colector, execution_date=20220821T000000, start_date=20220930T143143, end_date=20220930T143144
[2022-09-30T14:31:44.919+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 318 for task colector (Message: create container: Error response from daemon: No such image: selenoid/vnc:firefox_72.0
; 100)
[2022-09-30T14:31:44.931+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-30T14:31:44.959+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
